aries-site/src/layouts/content/Example.js

```diff
@@ -0,0 +1,164 @@
+    return () => clearTimeout(timer);
+  }, [hover]);
+
+  return (
+    <>
+      <Button
+        ref={ref}
+        hoverIndicator
+        onMouseEnter={() => setHover(true)}
+        onMouseLeave={() => setHover(false)}
+        title={hover ? undefined : title}
+        {...rest}
+      />
+      {hover && (
+        <Drop
+          target={ref.current}
+          align={{ bottom: 'top', right: 'right' }}
+          plain
+          stretch={false}
+        >
+          <Box
+            background={colors['background-contrast']}
+            border="bottom"
+            round="xsmall"
+            pad="small"
+          >
+            <Text truncate>{title}</Text>
+          </Box>
+        </Drop>
+      )}
+    </>
+  );
+};
+
+IconButton.propTypes = {
+  title: PropTypes.string,
+};
+
+export const Example = ({ code, docs, figma, ...rest }) => {
+  const theme = React.useContext(ThemeContext);
+  const [showCode, setShowCode] = React.useState();
+  const [codeText, setCodeText] = React.useState();
+  const [Syntax, setSyntax] = React.useState(syntax.dark);
+  const codeRef = React.useRef();
+
+  React.useEffect(() => {
+    if (showCode && !codeText) {
+      setCodeText('loading');
+      fetch(code)
+        .then(response => response.text())
+        .then(text => setCodeText(text));
+    } else if (showCode && codeText) {
+      Prism.highlightElement(codeRef.current);
+    }
+  }, [code, codeText, showCode, Syntax]);
+
+  // Set the Syntax component after highlightElement. This will cause
+  // highlightElement to be re-run when Sytanx changes. This is needed
+  // so the styling change is rendered.
+  React.useEffect(() => setSyntax(syntax[theme.dark ? 'dark' : 'light']), [
+    theme.dark,
+  ]);
+
+  return (
+    <Box margin={{ vertical: 'small' }}>
+      <Box
+        direction="row"
+        background={colors['background-front']}
+        pad="large"
+        {...rest}
+      />
+      {(code || docs || figma) && (
+        <Box direction="row" justify="end" gap="xsmall">
+          {figma && (
+            <IconButton
+              title="Design Templates"
+              icon={<Template />}
+              hoverIndicator
+              href={figma}
+              target="_blank"
+            />
+          )}
+          {docs && (
+            <IconButton
+              title="Component Properties"
+              icon={<Document />}
+              hoverIndicator
+              href={docs}
+              target="_blank"
+            />
+          )}
+          {code && (
+            <IconButton
+              title="Code"
+              icon={<Code />}
+              hoverIndicator
+              onClick={() => setShowCode(!showCode)}
+            />
+          )}
+        </Box>
+      )}
+      {showCode && (
+        <Box border background="background-contrast" pad="medium">
+          <Text size="xsmall" color="text">
+            <Syntax>
+              <code ref={codeRef} className="language-jsx">
```

Comment:
First time I see you are using `className` :)

Reply:
Yep, required by `prismjs`.